doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
+        If the user now gives the answer expected from the server, the response is accepted and the authentication is successful.
+
+        Multi Challenge is basically a chain of challenges. If, for example, a PIN is stored for a challenge response token, after
+        the first successful challenge, a second one is triggered, in which the PIN is expected as an answer.
+        If the PIN matches, the second challenge is also successful and the user is successfully authenticated.
+
+   Policy Conditions
+        Since privacyIDEA 3.1, :ref:`policy_conditions` allow to define more advanced rules
+        for policy matching, i.e. for determining which policies are valid for a
+        specific request.
+
+        Conditions can be added to a policy via the WebUI. In order for a policy to
+        take effect during the processing of a request, the request has to match not
+        only the ordinary policy attributes (see :ref:`policies`), but also *all*
+        additionally defined conditions that are currently active. If no active
+        conditions are defined, only the ordinary policy attributes are taken into
+        account.
+
+   Application Plugins
+        privacyIDEA comes with application plugins. These are plugins for
+        applications like PAM, OTRS, Apache2, FreeRADIUS, ownCloud, simpleSAMLphp
+        or Keycloak which enable these
+        application to authenticate users against privacyIDEA.
+
+        You may also write your own application plugin or connect your own application
+        to privacyIDEA. This is quite simple using a REST API
+        :ref:`rest_validate`. In order to support more sophisticated token types like
+        challenge-response or out-of-band tokens, you should take a look at the
+        various :ref:`authentication_modes`.
+
+   Subscription
+        privacyIDEA is enterprise software. Managing lots of authentication devices for lots of users is a task that occurs in a company network.
+        privacyIDEA is licensed under an Open Source license. This guarantees, that a company using the Open Source software privacyIDEA can use this
+        software for life. In contrast prorpietary software or software-as-a-service (SaaS) can be changed, billed differently or even completely deleted.
+        You could not do anything about it. The Open Source privacyIDEA is under your control – forever.
+        The Open Source license dos not mean that a company has no costs in regards to two factor authentication.
+        At least they need to pay the administrator.
+        In any case the Open Source license states that this software comes without any warranty. A company using privacyIDEA needs to be aware of this.
+
+        For the product privacyIDEA we provide the suitable `support <https://netknights.it/en/produkte/privacyidea/>`_ with a defined response time and with fixed costs.
+
```

Comment:
The paragraph is missing the last information, what the "subscription" is. A subscription for support and enterprise packaging.

Reply:
done in 6a521e1c8da886ba9e6f80c2c00ba8eff01eab42.